lumicks/pylake/nb_widgets/fd_selector.py

```diff
@@ -145,9 +162,10 @@ def update_plot(self, curve):
 
     @property
     def ranges(self):
+        """Return list of range timestamps"""
```

Comment:
Perhaps this can now be `range_timestamps` since it's not really `ranges` being returned

Reply:
This is part of the public API already and there are notebooks out that depend on this name. I could change it, but then we'd still need to keep `ranges` around with a deprecation notice.